api/compile.js

```diff
@@ -3,12 +3,14 @@ const pf = require('portfinder')
 const fileDB = require('../db/file-queries')
 const { spawn } = require('child_process')
 
+module.exports.sessionMap = new Map();
+
 module.exports.createConfigFile = username => {
   return pf.getPortPromise()
     .then(port => {
       module.exports.port = port
       const filename = `tmp/${username}_config`
-      const data = `port=${port}\njslibdir=/home/nick/.opam/4.04.0/lib/links/js\njsliburl=/lib/\ndatabase_driver=postgresql\ndatabase_args=localhost:5432:links:links`
+      const data = `port=${port}\njslibdir=/home/arek/.opam/4.06.0/lib/links/js\njsliburl=/lib/\ndatabase_driver=postgresql\ndatabase_args=localhost:5432:links:links`
```

Comment:
It would be good to make this configuration information read from an environment/config file somehow rather than hardwired...

Reply:
I would suggest using environment variables for this, like [this](https://github.com/links-lang/TryLinks-Server/blob/master/app.js#L30).